mobify/pinny
# Fix input disabling

Fixes the input disabling where using `data` caused it to not work!

Status: **Open for visibility**

Reviewers: @tedtate @kpeatt @Helen-Mobify 

## Changes
- fixed enabling/disabling inputs
- added checks to ensure these methods only run once if a pinny is open/lockup is locked.

## How to test-drive this PR
- checkout branch
- `grunt test`
- `grunt serve` and ensure that pinny is correctly enabling/disabling inputs on open/close

##TODO

- [ ] Write passing test for tab index storage on open
- [ ] Write passing test for tab index restoring on close
